src/email/create_user_analytics_dataset.R

```diff
@@ -0,0 +1,303 @@
+"
+Create user analytics data set.
+
+Script queries mailchimp API to get user analytics data and then combines it
+into a tidy data.frame.
+
+This will be run on a GitHub Action (.github/workflows/user_audience_analysis.yml)
+Weekly.
+"
+
+# putting this up here so that we can change to PROD later and only need
+# to update here rather than multiple places.
+FP_USER_INTERESTS = "output/user_research/hdx_signals_user_contacts.csv"
```

Comment:
`FP_USER_INTERESTS` and `FP_USER_CONTACTS` are swapped (:

Reply:
good call! - fixed